omod/src/main/webapp/pages/markPatientDead.gsp

```diff
@@ -127,7 +127,7 @@ ${ui.includeFragment("coreapps", "patientHeader", [patient: patient])}
                         formFieldName: "deathDate",
                         left         : true,
                         defaultDate  : patient?.getDeathDate() ?: defaultDeathDate ?: null,
-                        useTime      : false,
+                        useTime      : deceasedDateTimeComponent,
```

Comment:
The name `deceasedDateTimeComponent` does not look like a boolean variable. It sounds like an actual component.

Reply:
What do you think of `isDeceasedDateTime` ?

Reply:
How about something like? `includesTime`

Reply:
I will change to that.